test/move.js

```diff
@@ -0,0 +1,19 @@
+var R = require('../source');
+var eq = require('./shared/eq');
+
+describe.only('move', function() {
+  it('should correctly moves an element from an index to another', function() {
```

Comment:
Ramda usually skips "should" in the test description, and I don't see a good reason for "correctly" either.

```js
it('moves an element from one index to another', function() { /* ... */ })
```

seems cleaner to me.